src/Support/TypeToSchemaExtensions/ResponderTypeToSchema.php

```diff
@@ -0,0 +1,230 @@
+<?php
+
+namespace Dedoc\Scramble\Support\TypeToSchemaExtensions;
+
+use Dedoc\Scramble\Extensions\TypeToSchemaExtension;
+use Dedoc\Scramble\Support\Generator\Reference;
+use Dedoc\Scramble\Support\Generator\Response;
+use Dedoc\Scramble\Support\Generator\Schema;
+use Dedoc\Scramble\Support\Generator\Types as OpenApiTypes;
+use Dedoc\Scramble\Support\Generator\Types\UnknownType;
+use Dedoc\Scramble\Support\InferExtensions\ResourceCollectionTypeInfer;
+use Dedoc\Scramble\Support\Type\ArrayItemType_;
+use Dedoc\Scramble\Support\Type\ArrayType;
+use Dedoc\Scramble\Support\Type\Generic;
+use Dedoc\Scramble\Support\Type\Literal\LiteralBooleanType;
+use Dedoc\Scramble\Support\Type\ObjectType;
+use Dedoc\Scramble\Support\Type\Type;
+use Dedoc\Scramble\Support\Type\TypeWalker;
+use Dedoc\Scramble\Support\Type\Union;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Http\Resources\Json\AnonymousResourceCollection;
+use Illuminate\Http\Resources\Json\JsonResource;
+use Illuminate\Http\Resources\Json\ResourceCollection;
+use Illuminate\Http\Resources\MergeValue;
+use Illuminate\Http\Resources\MissingValue;
+use League\Fractal\TransformerAbstract;
+use Orchestra\Canvas\Commands\Database\Eloquent;
+
+class ResponderTypeToSchema extends TypeToSchemaExtension
+{
+    public function shouldHandle(Type $type)
+    {
+        return $type->isInstanceOf(TransformerAbstract::class);
+    }
+
+    /**
+     * @param  ObjectType  $type
+     */
```

Comment:
You're already type-hinting the param, you can remove the docblock.

Reply:
ObjectType extends Type and and the function comes from an implementation, so I have to use Type and ObjectType, both typings

Reply:
Oh, really? It doesn't work if you remove the docblock, even though `ObjectType` is a child of `Type`?